src/ServiceInsight/MessageList/MessageListViewModel.cs

```diff
@@ -245,6 +251,12 @@ void TryRebindMessageList(PagedResult<StoredMessage> pagedResult)
             }
         }
 
+        void ClearResult()
+        {
+            BindResult(new PagedResult<StoredMessage>());
+            SearchBar.ClearPaging();
+        }
+
```

Comment:
Is it worth creating this separate method instead of inlining in `Handle(ServiceControlConnectionChanged)`?